Library/Homebrew/test/support/lib/config.rb

```diff
@@ -15,21 +15,21 @@
 
 # Paths pointing into the Homebrew code base that persist across test runs
 HOMEBREW_LIBRARY_PATH  = Pathname.new(File.expand_path("../../../..", __FILE__))
-HOMEBREW_SHIMS_PATH    = HOMEBREW_LIBRARY_PATH.parent+"Homebrew/shims"
+HOMEBREW_SHIMS_PATH    = HOMEBREW_LIBRARY_PATH.parent.join("Homebrew", "shims")
```

Comment:
Ah, the `/` method is not available, because we load it after loading the test `config.rb`.